### PR TITLE
removes i18n repo reference [stage-6]

### PR DESCRIPTION
**Pull request by pcsandford**
@jungeunyoon @Rise-Vision/content please review - this removes the reference to the deprecated i18n repo and replaces them with the link to files now in common-header. Thanks!

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -35,12 +35,16 @@
     "rv-common-style": "https://github.com/Rise-Vision/common-style.git#1.3.57",
```
Comment (jungeunyoon) [inline bower.json]: @pcsandford I don't think you've removed `rv-common-i18n`

File: bower.json [hunk 2]
```diff
@@ -35,12 +35,16 @@
...
     "widget-common": "https://github.com/Rise-Vision/widget-common.git#3.4.10",
     "widget-settings-ui-components": "https://github.com/Rise-Vision/widget-settings-ui-components.git#3.2.2",
-    "widget-settings-ui-core": "https://github.com/Rise-Vision/widget-settings-ui-core.git#0.4.3"
+    "widget-settings-ui-core": "https://github.com/Rise-Vision/widget-settings-ui-core.git#0.4.3",
```
Comment (jungeunyoon) [inline bower.json]: @pcsandford Wondering how these additions (including lines 45-48) relate to removing references to deprecated `rv-common-i18n`?

Comment (pcsandford) [reply bower.json]: @jungeunyoon I am not sure - could it have happened with the node version? I realize I was using 8.8.0 as suggested to fix the test issue, but 'circle.yml' lists 6.9.1. Should I switch to that version and try again? A similar issue happened when working with common-header as the build would compile new CSS files when using a different node version. Thanks!

Comment (pcsandford) [reply bower.json]: @jungeunyoon actually, I think those were dependencies that I had to select when adding common-header - when it gives you the option of 2 or 3 versions and you have to select one, if that makes sense?

**Comment by pcsandford** [comment]
@jungeunyoon @Rise-Vision/content please review. I have removed the ref to the i18n repo in the bower.json file, and updated the README to refer to the common-header. The additional resolutions in the bower.json file are required by the common-header, otherwise the `gulp build` task fails. Thanks!

File: src/settings.html [hunk 3]
```diff
@@ -194,7 +194,7 @@ <h4>{{ "widget-web-page.interactivity.heading" | translate}}</h4>
   <!-- build:js js/settings.min.js -->
   <script src="components/angular-translate/angular-translate.js"></script>
   <script src="components/angular-translate-loader-static-files/angular-translate-loader-static-files.js"></script>
-  <script src="components/rv-common-i18n/dist/i18n.js"></script>
+  <script src="components/common-header/dist/js/components/i18n.js"></script>
```
Comment (pcsandford) [inline src/settings.html]: @jungeunyoon this is the reference I was referring to. Thanks!

**Pull request merged**